pkg/cmd/stack/root.go

```diff
@@ -213,6 +235,7 @@ func RootCommand() *cobra.Command {
 	cobra.CheckErr(stack.AddOptions(stackUpdateCmd, false))
 
 	stackCmd.AddCommand(stackDeleteCmd)
+	stackDeleteCmd.Flags().BoolVarP(&confirmDown, "yes", "y", false, "confirm the destruction of the stack")
```

Comment:
Would it be more intuitive to have the flag as --force/-f to destroy the stack unprompted?

Reply:
--force is probably better saved for highly destructive operations like deleting buckets containing files or databases.

Reply:
Can we make this confirmation a little more verbose - 

"Warning - This operation will destroy your stack, all deployed resources will be removed. Are you sure you want to proceed?"

If possible, a better approach might be to show the things being destroyed

"Warning - the following resources will be destroyed:"
     Type                 Name            Plan
 -   Stack           quickstart-dev  delete
 -   ├─ aws:s3:BucketObject  index.html       delete
 -   └─ aws:s3:Bucket        my-bucket           delete

Reply:
personally think the above is a bit excessive. pulumi doesn't do that do they? I think in general cli tools don't overly baby the user too much. I'll update the text tho'

Reply:
raised https://github.com/nitrictech/cli/issues/168
For now i'll update the text